Code/GraphMol/SubstructLibrary/SubstructLibrary.h

```diff
@@ -249,10 +249,15 @@ class RDKIT_SUBSTRUCTLIBRARY_EXPORT FPHolderBase {
     return rdcast<unsigned int>(fps.size() - 1);
   }
 
+  //! Adds a raw bit vector pointer to the fingerprinter (which takes ownership)
+  unsigned int addFingerprint(ExplicitBitVect *v) {
+    fps.push_back(v);
+    return rdcast<unsigned int>(fps.size() - 1);
+  }
+
   //! Adds a raw bit vector to the fingerprinter
   unsigned int addFingerprint(const ExplicitBitVect &v) {
```

Comment:
This needs a big caveat emptor here.  The fingerprint needs to be exactly the same as the one generated by makeFingerprint or bad things may happen at run time.